targetconfig.json

```diff
@@ -136,7 +136,9 @@
             "S07439-34681-01269-56955",
             "S83499-96784-35479-44416",
```

Comment:
Question who owns all these S* games?

Reply:
These are all games that the team has made. I added the same four that are on the multiplayer home page, plus one that I made and one that Kiki made.

Reply:
Can we reimport and export as the non S* links? S* games can change or get deleted by the user. So there is a risk of losing them.

Reply:
^^ Yeah, we should probably swap over for that + local caching

Reply:
Would it just be a matter of opening all the games and resharing to get a nonpersistent link then?
What are the first two that don't have 'S'?

Reply:
we don't need to clear out the existing ones, but we should swap them over. Yes, easiest is to open in an incognito window and reshare, then replace the links in the md & if possible also swap over the `targetConfig.multiplayer.games` ones further down in this file. First two are for asphodel follows directions so don't worry about those~